packages/app/package.json

```diff
@@ -10,8 +10,9 @@
     "test": "ts-node ../../run_tests.ts",
     "build": "npm run clean && npm run compile",
     "dev-build": "npm run clean && npm run dev-compile",
+    "dev-trans": "babel --plugins @babel/plugin-proposal-export-namespace-from ./dist/ --out-dir ./dist && babel --plugins @babel/plugin-transform-modules-commonjs ./dist/ --out-dir ./dist",
     "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
-    "dev-compile": "esbuild ./src/*.ts ./src/**/*.ts --tsconfig=tsconfig.json --outdir=dist && sh setup.sh",
+    "dev-compile": "esbuild ./src/*.ts ./src/**/*.ts --tsconfig=tsconfig.json --outdir=dist && npm run dev-trans && sh setup.sh",
```

Comment:
Shall we have a universal script for wrapping the dev-trans and dev-compile?

Reply:
For now, this `dev-trans` is just a temporary workaround. We will get rid of it eventually. In this case, I don't think we need to have a universal script.